compact.go

```diff
@@ -570,6 +574,10 @@ func (c *LeveledCompactor) write(dest string, meta *BlockMeta, blocks ...BlockRe
 		return errors.Wrap(err, "rename block dir")
 	}
 
+	elapsed := time.Since(start)
+	level.Info(c.logger).Log("msg", "writing blocks to disk finished")
```

Comment:
Let's put it them into single line. Easier to read and relate. 
```suggestion
	level.Debug(c.logger).Log("msg", "writing blocks to disk finished", "elapsed", time.Since(start))
```